india_compliance/gst_india/utils/e_waybill.py

```diff
@@ -1264,7 +1263,10 @@ def validate_applicability(self):
         self.validate_same_gstin()
 
     def validate_same_gstin(self):
-        if self.doc.doctype == "Delivery Note":
+        if self.doc.doctype == "Delivery Note" or (
+            self.doc.get("purpose") in ["Material Transfer", "Material Issue"]
+            and not self.doc.is_return
```

Comment:
possibility to use subcontracting order field

Reply:
Subcontracting order is not a mandatory field in Stock Entry and also is not mandatory for e-Waybill Generation.
So using it may not be a differentiating factor.